src/python/include/Euclidean_witness_complex_interface.h

```diff
@@ -47,11 +47,11 @@ class Euclidean_witness_complex_interface {
     delete witness_complex_;
   }
 
-  void create_simplex_tree(Gudhi::Simplex_tree<>* simplex_tree, double max_alpha_square, std::size_t limit_dimension) {
+  void create_simplex_tree(Simplex_tree<Simplex_tree_options_for_python>* simplex_tree, double max_alpha_square, std::size_t limit_dimension) {
```

Comment:
Is there a reason not to use `Simplex_tree_interface` here (and related places)?

Reply:
Here it is ok, so I changed it on 55f9f8e51
But there is an issue with Tangential_complex _interface while using `create_complex`. I added a comment on the problematic line